test/unit/lib/plugins/invoke.test.js

```diff
@@ -66,6 +66,13 @@ describe('Invoke', () => {
         );
       });
 
+      it("should accept everything after the first '='", () => {
```

Comment:
Thank you @terrybondy :bow: It looks good overall, however, we're currently in the process of migrating from older-style tests such as this one (and we've made a decision to not accept changes that are not using that approach to not add extra tech debt) to the ones based on `runServerless` utility as described here: https://github.com/serverless/serverless/blob/master/test/README.md

We've discussed that internally and it seems like the best bet would be to test out your case as a part of test that is specified here: https://github.com/serverless/serverless/blob/b81fd0557f53e11597fe67bb138c09675d61e0c5/test/unit/lib/plugins/aws/invokeLocal/index.test.js#L1543 

Probably the best way to test it would be to adjust the function to `console.log` the expected variable and assert on `stdoutData` that is being returned by `runServerless` invocation.

Please let me know what do you think

Reply:
Don't mean to hijack your PR by any means, but @pgrzesik is something like what I have in https://github.com/terrybondy/serverless/pull/1 what you're looking for?

Reply:
@lewgordon @pgrzesik this is just the sort of help I need today. Thanks!

Reply:
Hey @terrybondy - by no means I wanted to maneuver you into refactoring/working on tech debt, what's unfortunate in this case that we cannot really make exceptions when it comes to testing because each such fix, even if it's very simple, would introduce another piece of code (tests) that will need refactoring. However, the example shown by @lewgordon looks exactly like something that I had in mind :clap: I think we could move it/adjust it so it takes place of a test I've linked or we can keep it in `invoke.test.js` module, but in a separate `describe` to highlight that it's using `runServerless` approach. How would you like to proceed with wrapping up that PR @terrybondy @lewgordon ?

Reply:
@terrybondy, I'll leave it to you if you'd like. However, if you'd like assistance finishing up this test I don't mind working with you and @pgrzesik to do so!

Reply:
@lewgordon I would really appreciate assistance.

Reply:
@terrybondy, @pgrzesik I updated my PR. Sorry I misread your initial comment about the test placement. I think it should be fixed now, though. https://github.com/terrybondy/serverless/pull/1

Reply:
That looks great @lewgordon - @terrybondy, could you incorporate the changes from branch provided by @lewgordon to this PR? I believe we will be able to merge it at this point :+1:

Reply:
@lewgordon thanks for the assistance!

Reply:
Thanks gentlemen - unfortunately, I did local testing and in the current configuration, this test is not being executed at all - this is due to the fact that the whole `describe` in which it lives is skipped. I believe the best way to approach it here is to "unskip" the top-level describe, skip all `describes` for runtimes other than `Node.js` and then the test should be executed correctly. Sorry for the back and forth but it wasn't obvious from just looking at the code as I didn't realise that whole describe is skipped

Reply:
No worries! Good catch @pgrzesik. I had just ran the tests locally and assumed they passed. I created another PR against your fork @terrybondy: https://github.com/terrybondy/serverless/pull/2/files. @pgrzesik I needed to omit the production profile since it was giving me the following error:

```javascript
ServerlessError: AWS profile "production" doesn't seem to be configured
```

I wasn't sure if that's something that was easily fixable on my end. Let me know your thoughts.

Reply:
Thanks @lewgordon - I think it's fine to omit that profile for now and we'll revisit it during implementation of these tests. Thanks to both of you :bow: